examples/poisson_3d_target_torus.py

```diff
@@ -159,9 +159,9 @@ def run_model(ncells, degree, comm=None, is_logical=False):
 
     # Create discrete spline space
     if is_logical:
-        Vh = discretize(V, Omega_log_h, degree=degree, periodic=[False, True, True])
+        Vh = discretize(V, Omega_log_h, degree=degree)
```

Comment:
@yguclu here I did not know where to put the periodicity information correctly, so I removed it in both cases. Unless already included somewhere else, this change will yield wrong results.

Reply:
The periodicity information should be provided to the discretized domain, i.e. when you create `Omega_log_h` (see function [discretize_domain](https://github.com/pyccel/psydac/blob/566c071fcf8f7c35afe542147d360f7acca313ff/psydac/api/discretization.py#L370-L386)).